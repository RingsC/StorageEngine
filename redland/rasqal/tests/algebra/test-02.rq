# http://www.w3.org/TR/2008/REC-rdf-sparql-query-20080115/#convertGraphPattern
#
# Example: group consisting of a union of two basic graph patterns:
#
PREFIX : <http://example.org#>
SELECT * WHERE
{ ?s :p1 ?v1 ; :p2 ?v2 }
