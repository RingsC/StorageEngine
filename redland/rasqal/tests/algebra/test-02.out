Project(
        BGP(
            triple(variable(s), uri<http://example.org#p1>, variable(v1)) ,
            triple(variable(s), uri<http://example.org#p2>, variable(v2))
        ) ,
        Variables([ variable(s), variable(v1), variable(v2) ])
)
